java/src/main/java/com/plaid/quickstart/QuickstartConfiguration.java

```diff
@@ -21,6 +21,20 @@
   @NotEmpty
   private String plaidCountryCodes;
 
+  // Parameters used for the OAuth redirect Link flow.
+
+  // Set PLAID_OAUTH_REDIRECT_URI to 'http://localhost:8000/oauth-response.html'
+  // The OAuth redirect flow requires an endpoint on the developer's website
+  // that the bank website should redirect to. You will need to whitelist
+  // this redirect URI for your client ID through the Plaid developer dashboard
+  // at https://dashboard.plaid.com/team/api.
+  private String plaidOauthRedirectUri;
```

Comment:
Does this need to be `Oauth` for the JSON deserialization, or could we `s/Oauth/OAuth/g`?

Reply:
It does not need to be Oauth, OAuth should be better https://github.com/plaid/quickstart/pull/97/commits/e357d4baee95f7fa1c1379658a870560cf71bf1f